pennylane/compiler/qjit_api.py

```diff
@@ -430,14 +460,10 @@ def for_loop(lower_bound, upper_bound, step, loop_fn, *args):
         across iterations is handled automatically by the provided loop bounds, it must not be
         returned from the function.
 
-    Raises:
-        CompileError: if the compiler is not installed
-
     .. seealso:: :func:`~.while_loop`, :func:`~.qjit`
 
     **Example**
 
-
```

Comment:
Do we still need to keep the `qml.qjit` in the code example? I'd probably recommend getting rid of it, and then adding a mention that "this works with qjit".

Reply:
Good call, I updated the docstring here: https://github.com/PennyLaneAI/pennylane/pull/6014/files/ee2532242b9d6da4f6e1c37125519e9fecf94a5b..f833dd33c64aeb8a2589b53b8ea0e3c7f8e2d509